analysis/shaman/src/Maelstrom.tsx

```diff
@@ -13,7 +15,7 @@ const COLORS = {
   WASTED_MAELSTROM_BORDER: 'rgba(255, 90, 160, 1)',
 };
 
-const Maelstrom = props => {
+const Maelstrom = (props: any) => {
```

Comment:
Instead of any, would it be better to declare an interface? I believe since there is a propTypes declared on line 114 you can just declare an interface there instead and expect the props be be of the Maelstrom.